.github/workflows/release.yml

```
@@ -27,4 +27,24 @@ jobs:
           push: true
           tags: |
             ${{ env.docker_image }}:latest
-            ${{ env.docker_image }}:${{ github.event.release.tag_name }}
\ No newline at end of file
+            ${{ env.docker_image }}:${{ github.event.release.tag_name }}
+
+
+  build_binary:    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)